legacy/app/common/global/event-handlers.js

```diff
@@ -14,4 +14,6 @@ function (
     $rootScope.toggleModalVisible = function (state) {
             $rootScope.modalVisible = (typeof state !== 'undefined') ? state : !$rootScope.modalVisible;
     };
+
+    $rootScope.sources = ['sms', 'twitter','web', 'email', 'whatsapp', 'ussd'];
```

Comment:
@Ifycode as one last thing to do before we can merge this to trunk branches...

Could this variable come from config.js ? Something like 'enabled_sources' that defaults to SMS, twitter, web and email.

That way we could keep away whatsapp and ussd for now, and only enable them in the few deployments where they are actually used.

Reply:
All right @tuxpiper. I will check out how to add it to config.